develop.yml

```
@@ -18,7 +18,7 @@ services:
       - mongo
   back:
     build:
-      context: "https://github.com/camicroscope/caracal.git#develop"
+     context: "https://github.com/camicroscope/caracal.git#develop"
```

compose has complained about this missing space in the yml
```suggestion
      context: "https://github.com/camicroscope/caracal.git#develop"
```